web/package.json

```diff
@@ -17,7 +17,6 @@
     "react-dom": "^17.0.2",
     "react-scripts": "5.0.0",
     "sass": "^1.49.0",
-    "use-root-reducer": "^1.0.1",
```

Comment:
This closes https://github.com/yast/the-installer/issues/13, which makes me happy :stuck_out_tongue_closed_eyes: